File: findHypotenuse.cpp
#include<iostream>
#include<cmath>
using namespace std;
int main(){
	int x, y;
	double hypotenuse;
	cout << "Enter the corner = ";
	cin >> x >> y;
	hypotenuse = sqrt(pow(x,2) + pow(y,2));
	cout << "Hypotenuse = " << hypotenuse;
	return 0;
}

File: findHypotenuse.java
package springIntro;
import java.util.*;
public class findHypotenuse {
	public static void main(String[] args) {
		Scanner Input = new Scanner(System.in);
		System.out.println("Please enter the corners");
		int x = Input.nextInt();
		int y = Input.nextInt();
		double hypotenuse = Math.sqrt(Math.pow(x, 2) + Math.pow(y, 2));
		System.out.println("Hypotenuse = " + hypotenuse);
	}
}
